tests/test_sdrf.py

```
@@ -3,39 +1,40 @@
 from quantmsio.core.sdrf import SDRFHandler
-
 from .common import datafile
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)